src/smart-components/Thread/components/ThreadHeader/index.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../../../styles/variables';
+
+.sendbird-thread-header {
+  position: relative;
+  min-width: 320px;
+  width: 320px;
+  min-height: 64px;
+  height: 64px;
+
+  display: inline-flex;
+  flex-direction: column;
+  padding: 13px 24px 13px 24px;
```

Comment:
There is padding shorthands, if you are interested in those things ;) https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#margin_and_padding_properties